tests/e2e/pact/specs/live/eps.convert.spec.ts

```diff
@@ -26,10 +27,12 @@ jestpact.pactWith(
       test(authenticationTestDescription, async () => {
         const apiPath = "/$convert"
         const interaction: InteractionObject = createUnauthorisedInteraction(authenticationTestDescription, apiPath)
+        const requestId = uuid.v4().toString().toLowerCase()
         await provider.addInteraction(interaction)
         await client()
           .post(apiPath)
           .set('Content-Type', 'application/fhir+json; fhirVersion=4.0')
+          .set('X-Request-ID', requestId)
```

Comment:
could move this to requestFilter so it's only set in one place, see verify.ts

Reply:
unless we're checking the response guid too as suggested by Matt below, in which case here is fine

Reply:
We are checking the X-Request-ID header matches in the request and response, I'll add in the check for X-Correlation-ID too. Is there a nice way to centralise doing those checks or would it require a lot of refactoring?

Reply:
Could look at wrapping that client in a function inside the common.ts Dan's just added